### PR TITLE
Moved terms of use logic to a separate class

### PR DESCRIPTION
**Pull request by Wiezzel**
As suggested in https://github.com/golemfactory/golem/pull/2373#discussion_r176162100

### CODE REVIEW

File: golem/node.py [hunk 1]
```diff
@@ -284,3 +277,27 @@ def _stop_on_error(self, msg: str, err: Any) -> None:
         if self._reactor.running:
             logger.error("Stopping because of %r error: %r", msg, err)
             self._reactor.callFromThread(self._reactor.stop)
+
+
+class TermsOfUse:
```
Comment (Krigpl) [inline golem/node.py]: Having it in a separate file would also be fine

**Review by Krigpl — APPROVED** [review]
verynicegreatsuccess

File: golem/node.py (hunk 1 above)
Comment (Wiezzel) [reply golem/node.py]: In the same directory?

Comment (Krigpl) [reply golem/node.py]: Yeah

Comment (Wiezzel) [reply golem/node.py]: Voilà.

**Pull request merged**